src/pages/dataTableUtilities/getColumns.js

```diff
@@ -367,6 +374,14 @@ const COLUMNS = () => ({
     sortable: false,
     editable: false,
   },
+  [COLUMN_NAMES.INVOICE_TYPE]: {
+    type: COLUMN_TYPES.STRING,
+    key: COLUMN_KEYS.INVOICE_TYPE,
+    title: tableStrings.type,
+    alignText: 'left',
+    sortable: true,
+    editable: false,
+  },
```

Comment:
This is just below a `TRANSACTION_TYPE` column definition !

Reply:
Yeah, I added this as the `TRANSACTION_TYPE` key doesn't seem to be pretty printed (e.g. `supplier_invoice`). I can see now there is a `formatType` helper which does this inside the data table row logic which substitutes the type for a localised string.

Will update to use the existing logic but I think this might be a small example of where the data table could be simplified. I think it's a bit of a potential smell that certain keys fire conditionals within the data table and are processed/formatted in a special way. My thinking here is that this would be better as a helper on the `Transaction` type (like I have here as `invoiceType`, admittedly not a great name). That way it would a be more explicit and wouldn't bind the view (the data table) as closely to the underlying model (the row object). I know there are probably other examples which make this only a small thing, but wherever it's possible to loosen the binding between the data table and the data types, I think it's probably a good idea?